src/main/java/org/prebid/server/bidder/mobilefuse/MobilefuseBidder.java

```diff
@@ -46,26 +47,30 @@ public MobilefuseBidder(String endpointUrl, JacksonMapper mapper) {
 
     @Override
     public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
-        final ExtImpMobilefuse firstExtImpMobilefuse = request.getImp().stream()
+        final Optional<ExtImpMobilefuse> firstExtImpMobilefuse = request.getImp().stream()
```

Comment:
i'd say we don't strongly require to involve `Optional` here. It is just usual checking for NULL.
So, pls align it with our project style.